package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "axios": "^0.18.0",
```

Comment:
no need to install axios, you have already fetch. It's ok, but remember about it

Reply:
I read once that using axios is a safer approach